hi-ml/src/health_ml/experiment_config.py

```diff
@@ -0,0 +1,16 @@
+import param
+from typing import Any
+
+from health_azure.utils import GenericConfig
```

Comment:
do we want to keep this GenericConfig class? We mainly used it to auto-generate argument parsers, but that can be done in a simpler way too - just check if the class in questions derives from params.Parameterized